oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1015,7 +1018,7 @@ private String getConsoleUrl(String cluster, String actionId) throws FalconExcep
             // Get conf when workflow is launched.
             if (coordinatorAction.getExternalId() != null) {
                 WorkflowJob jobInfo = client.getJobInfo(coordinatorAction.getExternalId());
-
+                workflowUser = jobInfo.getUser();
```

Comment:
Interesting. Does the message contain the correct workflow user?
From the log, the user for rerun used to be "falcon". Balu's change corrects it.